community/cypher/cypher-compiler-2.2/src/main/scala/org/neo4j/cypher/internal/compiler/v2_2/ast/rewriters/CNFNormalizer.scala

```diff
@@ -19,23 +19,22 @@
  */
 package org.neo4j.cypher.internal.compiler.v2_2.ast.rewriters
 
-import org.neo4j.cypher.internal.compiler.v2_2._
 import org.neo4j.cypher.internal.compiler.v2_2.ast._
-import org.neo4j.cypher.internal.compiler.v2_2.repeat
+import org.neo4j.cypher.internal.compiler.v2_2.{repeat, _}
 
-case object CNFNormalizer extends Rewriter {
+case class CNFNormalizer()(implicit monitor: AstRewritingMonitor) extends Rewriter {
```

Comment:
Why do we take the monitor as an implicit? It looks like when we actually call it we always provide it explicity, so in that case we might as well have 
```case class CNFNormalizer(implicit monitor: AstRewritingMonitor) extends Rewriter```